docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -329,6 +329,11 @@ content: |
   Self-described gender
 ---
 generic object: ALIndividual
+template: x.gender_unknown_label
+content: |
+  Unknown
+---
+generic object: ALIndividual
```

Comment:
I know this is not part of the changes you made, but `x.gender_female_label` seems to be defined twice.